packages/compass-e2e-tests/helpers/commands/sidebar-connection.ts

```diff
@@ -0,0 +1,113 @@
+import { TEST_MULTIPLE_CONNECTIONS } from '../compass';
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function getConnectionIdByName(
+  browser: CompassBrowser,
+  connectionName: string
+): Promise<string> {
+  // NOTE: this is only supported for multiple connections
+  return await browser
+    .$(Selectors.sidebarConnection(connectionName))
+    .getAttribute('data-connection-id');
+}
```

Comment:
given that we do not enforce unique names, this is a bit risky 🤔 . could we at least check if there are multiple and throw in that case? so that we don't end up with some weird test behaviour on accident.